openshift/ci-operator/knative-images/kn/Dockerfile

```diff
@@ -1,6 +1,8 @@
 # DO NOT EDIT! Generated Dockerfile for cmd/kn.
 ARG GO_BUILDER=registry.ci.openshift.org/openshift/release:rhel-8-release-golang-1.22-openshift-4.17
 ARG GO_RUNTIME=registry.access.redhat.com/ubi8/ubi-minimal
+ARG IMAGE_KN_PLUGIN_FUNC_UTIL=registry.redhat.io/openshift-serverless-1/kn-plugin-func-func-util-rhel8sha256:49087b51e8b97d30ff7db67e1983d4ee63665092009af20a2598d7410eb781fb
```

Comment:
Where will these images come from for 1.35? I guess we will need brew.registry.redhat.io because the image won't be public yet?
In that case we need a public image again (like in my previous comment) and for Konflux we'll override it via ARGS with the brew.registry.redhat.io variant.

Reply:
Ideally, both of those images should come from nudges with updated reference per pipeline build.

Reply:
I'm not sure how/if the nudges will work here. For this the image should point to an image from Konflux, IMO.

Reply:
In theory this image doesn't have to be pull-able during build time. For the build time we need final sha. At least that  how we used to build it internally.

Reply:
Discussed on slack.